qiime2/metadata/metadata.py

```diff
@@ -1083,7 +1035,11 @@ def __eq__(self, other):
         return (
             super().__eq__(other) and
             self.name == other.name and
-            self._series.equals(other._series)
+            self._series.equals(other._series) and
+            # My initial inclination was to make this test for equalty of
+            # _source_metadata, but this seriously messes with some of the
+            # rountrip tests in test_io, so it looks like this for now
+            self.artifacts == other.artifacts
```

Comment:
Again, same as comment. Do we want this to test for equality of `artifacts` or equality of `_source_metadata`.?

Reply:
Hmm, let's stick with this at the moment, but please leave a `TODO` note for us to revisit before merging.